### DISCUSSION TITLE
Security Onion - Not seeing Alerts nor Hunt, but getting alerts in Kibana

### DISCUSSION THREAD

**Discussion by jacekradomski839** [Security-Onion-Solutions/securityonion#6082] [Unsupported Versions] [answered]
I have multiple sensors and a Manager/Search mode device. Logs are not visible in Alerts and Hunt in the SOC, but I can see the alerts in the Kibana and Grid section indicating increments on the EPS (Events Per Second). Initially, I had run ver 2.3.40 and I had read that upgrading to 2.3.80 might resolve the issue, but it didn't help.  Someone suggested changing the Time in Hunt to Absolut Time might fix it, but nothing. I had checked all services and they are also running. Any ideas?

**Comment by tiger-greenhell** [comment, marked as answer]
#5920
